templates/aanmelder/onderdelen/reeks_activiteiten.html.twig

```diff
@@ -0,0 +1,31 @@
+<H3>
+	{{ reeks.naam }}
+</H3>
+<p>
+	{{ reeks.rawBeschrijving | bbcode }}
+</p>
+
+{% if activiteiten|length %}
```

Comment:
Deze mist nog wat css... Hij zou in ieder geval over de hele breedte kunnen spannen.
Maar dat is misschien een beetje mijn eigen nalatenheid...

Reply:
Yarn runnen :D